webform_prefill.module

```diff
@@ -44,15 +48,16 @@ function webform_prefill_libraries_info() {
 }
 
 /**
- * Implements hook_form_FORM_ID_alter().
```

Comment:
Coding-style would be `Implements hook_form_FORM_ID_alter() for webform_client_form().`